Main.java

```diff
@@ -1,3 +1,5 @@
+import java.util.Scanner;
+
 /**
  *
  * @author 

```

Comment:
Make sure that you have your name and the program description in the code header!